templates/csharp/app-mvvm/AvaloniaAppTemplate.csproj

```diff
@@ -15,17 +15,16 @@
     <AvaloniaResource Include="Assets\**" />
   </ItemGroup>
 
-  <!--#if (AvaloniaStableChosen) -->
+  <!--#if (!AvaloniaStableChosen) -->
```

Comment:
Feels like "stable" is not a valid term here, as both version are stable at this point. Some kind of enum property or multiple bool properties for each version might work better.

Reply:
I discussed with @Takoooooo about this and suggested "UseCurrentVersion",  he meant "lets leave AvaloniaStableChosen,change it to AvaloniaVersion == \"11.0.0\" and negate expressions where it was used".

But agree enum like would be the easiest to maintain, as it would not change meaning anymore. Something like "IsV11" and "IsV0_10"?

Reply:
"IsV10" should be fine too :) 
But yes, we can keep it out of this PR